content/07.Publication_Process.md

```diff
@@ -1,15 +1,38 @@
 # The Publication and Peer Review Process 
-
 <!-- Leader: Victoria Nolan; co-leaders: Sprih Harsh --> 
 
+Section outline: This section aims to provide an overview of the research publication and peer review process, both from the perspective of a researcher submitting a manuscript to a journal and from that of the peer reviewer providing comments on the work. This content is aimed at students and early-career researchers who may have no or little knowledge of the publication process, or how to become a successful and good peer-reviewer. It will cover the follow topics:
+
 ## Determining coauthorship 
+- Criteria to determine if a researcher, student, technician etc. qualifies for author contributions 
+- General journal threshold requirements for authorship and how to write a contributions statement
+- Determining the order of the authors
+- Acting as the corresponding author – commitments and duties
```

Comment:
I would include the Credit approach as a sound way to account for authors' contributions.